x/stake/tick_test.go

```diff
@@ -1,13 +1,30 @@
 package stake
 
 import (
+	"fmt"
+	"strconv"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	candidateAddrs = []sdk.Address{
+		addrs[0],
+		addrs[1],
+		addrs[2],
+		addrs[3],
+		addrs[4],
+		addrs[5],
+		addrs[6],
+		addrs[7],
+		addrs[8],
+		addrs[9],
```

Comment:
I don't think we need to define/use `candidateAddrs` when we can just use `addrs` directly

Reply:
yes , no need to make candidateAddrs

Reply:
fixed